src/test/java/org/dom4j/datatype/SchemaParseTest.java

```diff
@@ -14,7 +14,7 @@
 import org.dom4j.io.SAXReader;
 
 /**
- * Tests the SchemaParser based on a test case provided by Luis Pe�a S�nchez
+ * Tests the SchemaParser based on a test case provided by Luis Peta Sanchez
```

Comment:
?

Reply:
I'll remove this comment altogether as adds no value